### PR TITLE
Hindi language support added

### LINKED ISSUE #67: [SUGGESTION] add Hindi Language support

### PR DESCRIPTION
**Pull request by Vaibhav-Magadum**
@goldyfruit @suvanbanerjee kindly review
closes #67

### CODE REVIEW

File: tui/locales/hi-in/profiles.sh
Comment (goldyfruit) [inline tui/locales/hi-in/profiles.sh:1]: You are missing the `server` profile (added yesterday :+1: ).

https://github.com/OpenVoiceOS/ovos-installer/blob/main/tui/locales/en-us/profiles.sh#L9

**Comment by goldyfruit** [comment]
From a code point of view it looks OK but I can't review Hindi :rofl: 

@suvanbanerjee could you please have a quick look as a second pair of eyes?

**Comment by suvanbanerjee** [comment]
@goldyfruit Sure :)

**Comment by suvanbanerjee** [comment]
@goldyfruit LGTM 👍

**Comment by goldyfruit** [comment]
Thanks @suvanbanerjee @Vaibhav-Magadum!!

**Pull request merged**